src/pybind/mgr/mgr_module.py

```diff
@@ -1027,7 +1021,12 @@ def get_store(self, key, default=None):
             return r
 
     def get_localized_store(self, key, default=None):
-        return self._get_localized(key, default, self.get_store)
+        r = self._ceph_get_store('{}/{}'.format(self.get_mgr_id(), key))
+        if r is None:
+            r = self._ceph_get_store(key)
+        if r is None:
```

Comment:
Another über-minor thing, but in this case I'd nest the second `if r is None`, as it only makes sense within the previous if branch.